v8/plantuml/README.md

```diff
@@ -0,0 +1,30 @@
+This plugin converts [PlantUML](https://plantuml.com/) files.
+
+The default configuration will output all `*.puml` files found under the `pages` dir as SVG files.
```

Comment:
Good point.  The long term thing I'm working on is using Nikola for developer docs, in a recent work project we had quite a few sequence diagrams and after trying a few different options I found it easiest to keep the plant files in the same dir as the related markdown files.  Guess that idea has leaked over to here but you're right, the default should behave as a simpler generic website, perhaps this?

```python
PLANTUML_FILES = (
  ('plantuml/*.puml', 'plantuml', '.svg', ['-tsvg']),`
)
```

Reply:
This would be a better definition. Putting files that are not meant to be rendered as HTML in `pages` is not recommended (it works, but it’s an ugly choice).